certified-connectors/PPM Express/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2953 @@
+      }
+    },
+    "task_id__in_path": {
+      "in": "path",
+      "name": "taskId",
+      "required": true,
+      "type": "string",
+      "format": "uuid",
+      "x-ms-summary": "Task Id",
+      "x-ms-url-encoding": "single",
+      "x-ms-visibility": "important",
+      "description": "The unique identifier of the task",
+      "x-ms-dynamic-values": {
+        "operationId": "ProjectsTasks_GetAll",
+        "parameters": {
+          "id": {
+            "parameter": "id"
+          }
+        },
+        "value-path": "id",
+        "value-title": "attributes/Name",
+        "value-collection": "data"            
+      }
+    },
+      "resource_id__in_path": {
+        "in": "path",
+        "name": "id",
+        "required": true,
+        "type": "string",
+        "format": "uuid",
+        "x-ms-summary": "Resource Id",
+        "x-ms-url-encoding": "single",
+        "x-ms-visibility": "important",
+        "description": "The unique identifier of the resource",
+        "x-ms-dynamic-values": {
+          "operationId": "Resources_GetAll",
+          "value-path": "id",
+          "value-title": "attributes/Name",
+          "value-collection": "data"
+        }
+      }    
+  },
+  "responses": {},
+  "securityDefinitions": {
+    "API Key": {
+      "type": "apiKey",
+      "in": "header",
+      "name": "Authorization"
+    }
+  },
+  "security": [
+    {
+      "API Key": []
+    }
+  ],
+  "tags": [
+    {
+      "name": "Fields",
+      "description": "This resource represents fields for PPM Express Entities. Use it to obtain field details."
+    },
+    {
+      "name": "Challenges",
+      "description": "This resource represents PPM Express challenges. Use it to create, and update challenges as well as obtain a list of all challenges."
+    },
+    {
+      "name": "Ideas",
+      "description": "This resource represents PPM Express ideas. Use it to create, and update ideas as well as obtain a list of all ideas."
+    },
+    {
+      "name": "Me",
+      "description": "This resource represents current PPM Express user information."
+    },
+    {
+      "name": "Projects",
+      "description": "This resource represents PPM Express projects. Use this resource to create and update projects."
+    },
+    {
+      "name": "ProjectsKeyDates",
+      "description": "This resource represents Key Dates. Use it to create, and update Key Dates as well as obtain a list of all Key Dates."
+    },
+    {
+      "name": "ProjectsTasks",
+      "description": "This resource represents PPM Express tasks. Use it to create, and update tasks as well as obtain a list of all tasks."
+    },
+    {
+      "name": "Resources",
+      "description": "This resource represents PPM Express resources. Use it to create, and update resources as well as obtain a list of all resources."
+    },
+    {
+      "name": "Webhooks",
+      "description": "This resource represents Webhooks. Use it to create and delte webhooks to PPM Express."

```

Comment:
@vsorokin-ppmexpress please make sure that everything is grammatically correct and is free of typographical errors. Refer to this [guideline](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).